hardware/areca.py

```diff
@@ -147,6 +150,13 @@ def _disable_password():
 
 def detect():
     """Detect Areca controller configuration."""
+    if not detect_utils.which('cli64'):
+        sys.stderr.write('Cannot find cli64 binary\n')
+        return []
+    detect_areca()
```

Comment:
Damn, sorry, I should stop spotting an issue at a time... You probably want a return here.

Reply:
/facepalm yeah...